src/tiledb/cloud/bioimg/helpers.py

```diff
@@ -24,19 +24,25 @@ def iter_paths(sequence) -> Iterator[Tuple]:
             if uri.endswith((".tiff", ".tif", ".tdb")):
                 yield uri, create_output_path(uri, output_dir)
 
-    if len(source) == 1 and vfs.is_dir(source[0]):
-        # Check if the dir is actually a tiledb group for exportation
-        with tiledb.scope_ctx(ctx_or_config=config):
-            if tiledb.object_type(source[0]) != "group":
-                # Folder like input
-                contents = vfs.ls(source[0])
-                if len(contents) != 0:
-                    return tuple(iter_paths(contents))
+    if len(source) == 1:
+        if source[0].startswith("tiledb://"):
+            # Support tiledb uri single image
+            return ((source[0], create_output_path(source[0], output_dir)),)
+        elif vfs.is_dir(source[0]):
```

Comment:
Instead of checking for `is_dir` why not check for the object type?

Reply:
I check for the object type 2 locs below that one, but I wanted to validate arguments. Normally I should have add an error. In case no tiledb uri neither a vfs dir is given as input. Imagine a random single s3 blob as input, this should fail. Now it is  a no-op but maybe should throw an error.

Reply:
Also we support the case that the input is a folder of bioimg groups. This is why we check if it is single dir and if so we iterate to find the contents.